src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/PremisesService.kt

```diff
@@ -91,4 +95,74 @@ class PremisesService(
 
       return success(lostBedsEntity)
     }
+
+  fun createNewPremises(requestBody: NewPremises) = validated<PremisesEntity> {
+
+    // TODO: dummy data for now - remove asap
```

Comment:
Are `ApArea` and `ProbationRegion` populated like this because CAS3 Premises don't follow the same organisational heirarchy as CAS1 Premises?

Reply:
Yeah it's because, for now, only local authority is passed through from the FE when creating a new premises. If there's a better default way to address this I'd be interested to know about that

Reply:
I this because 'ApArea' and 'ProbationRegion' aren't relevant to a CAS3 premises though?  i.e. they will never be passed from the Frontend because they're not properties a CAS3 premises has?

Reply:
At the moment yeah but I can't say for definite if they won't be needed at some point

Reply:
Does your product owner or a domain expert from CAS3 perhaps know?  Keen that we make anything that isn't actually relevant for both types of premises is nullable rather than just filled with placeholder data